openwisp-monitoring/files/monitoring.agent

```diff
@@ -195,6 +197,11 @@ send_data() {
 			done
 			# retry sending same data again in next cycle
 			[ "$failures" -eq "$MAX_RETRIES" ] && break
+            # pause for a while after every 10 successful request sent
+			if [ $((success_count % 10)) -eq 0 ]; then
```

Comment:
Isn't `0 % 10 == 0` ? Why not just use `[ $success_rate -ge 10 ]` which is more readable?

Reply:
`-ge` makes it run whenever the `success_rate` is more than or equal to 10. But as per the issue, it was meant for every 10 requests. Was I right on this one?

Reply:
The requirement is to pause every 10 successful requests.
I don't see any `success_rate` variable in this diff, @pandafy where is `success_rate` coming from?
I only see `success_count` here.